<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaviWidget2DClass</class>
 <widget class="QWidget" name="NaviWidget2DClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>532</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>140</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>140</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVolume">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>113</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>12</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>12</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnControl">
   <property name="geometry">
    <rect>
     <x>492</x>
     <y>31</y>
     <width>24</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>12</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>12</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="disable_mask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>532</width>
     <height>140</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="qMRMLColorTableComboBox" name="ColorTableComboBox" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>20</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblName">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>图像名CT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="qMRMLVolumePropertyNodeWidget" name="VolumePropertyNodeWidget" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>511</width>
     <height>224</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="qSlicerVolumeRenderingPresetComboBox" name="PresetComboBox" native="true">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>162</y>
     <width>350</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ROI</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkVolume">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>容积再现</string>
   </property>
  </widget>
  <zorder>lblImage</zorder>
  <zorder>btnVolume</zorder>
  <zorder>btnControl</zorder>
  <zorder>ColorTableComboBox</zorder>
  <zorder>checkBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>VolumePropertyNodeWidget</zorder>
  <zorder>PresetComboBox</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>chkVolume</zorder>
  <zorder>disable_mask</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLVolumePropertyNodeWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLVolumePropertyNodeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerVolumeRenderingPresetComboBox</class>
   <extends>qSlicerWidget</extends>
   <header>qSlicerVolumeRenderingPresetComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLColorTableComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLColorTableComboBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
